### PR TITLE
Document aCL.threadId and notification.threadId

### PR DESCRIPTION
**Pull request by ghost**
Also:
- Add a missing requirement on URI base id properties.
- Fix a formatting typo

@michaelcfanning .

### CODE REVIEW

File: document/FileFormat.mdk [hunk 1]
```diff
@@ -3023,6 +3029,11 @@ or is such that the analysis might be incomplete but the results that were gener
 
 If the `level` property is absent, it shall be considered equivalent to the value `"warning"`.
 
+### `threadId` property { #notification-threadId }
+
+A `notification` object may contain a property named `threadId` whose value is an integer which
+identifies the thread which generated this notification.
```
Comment (michaelcfanning) [inline document/FileFormat.mdk:3035]: the thread 'associated' with this notification is better. right? the thread generating the notification could be a dedicated thread that's processing error messages produced across the app.

**Comment by michaelcfanning** [comment]
+1 modulo comment about 'associated' thread id  not 'generated'. maybe you could scan for this distinction elsewhere in the spec.

File: document/FileFormat.mdk (hunk 1 above)
Comment (ghost) [inline document/FileFormat.mdk:3035]: Yes, fixed.

**Comment by ghost** [comment]
I looked for other occurrences of "generated" and "associated" and didn't find any similar problems.

**Comment by ghost** [comment]
Oops. Merged before actually pushing the fix. Will push fix and merge again.

**Pull request merged**